docs/source/release_notes/release-notes.rst

```diff
@@ -50,6 +50,10 @@ develop
     *    - Type
          - Change
 
+    *    - |devbreak|
```

Comment:
where in the release notes should this go? should I backport the release notes change at all?

Reply:
Hmm, yeah we historically haven't maintained release notes properly for backport branches. Putting it under develop should be fine.